dist/obs-server.spec

```diff
@@ -931,7 +931,6 @@ usermod -a -G docker obsservicerun
 %{__obs_api_prefix}/db/data
 %{__obs_api_prefix}/db/migrate
 %{__obs_api_prefix}/db/seeds.rb
-%{__obs_api_prefix}/files/wizardtemplate.spec
```

Comment:
Breaks the package build...

Reply:
The package build is failing due to those errors:
```
RPM build errors:
     File listed twice: /usr/lib/obs/server/BSConfig.pm
     Directory not found: /home/abuild/rpmbuild/BUILDROOT/obs-server-2.11~alpha.20200901T093837.fab91c9552-2.1.x86_64/srv/www/obs/api/files
```

Reply:
Yeah, you remove `files` directory but we explicitly package that somewhere...

Reply:
Oh right, I forgot to remove that directory from the spec file.